.gitignore

```diff
@@ -113,3 +113,6 @@ service-definitions.yml
 
 # Rendered API docs
 /api-docs/
+
+# Flamebearer
+flamegraph.html
```

Comment:
I think you also want to gitignore `flamegraph.png` and remove it from this PR

Reply:
I think it's being linked to from docs. If we need to check this in for doc purposes but don't want it in master, another option is the solution I outlined here: https://stackoverflow.com/a/42677655/893113. I don't feel strongly either way.

Reply:
oh yes - sorry. I hasn't clocked it is used in the docs